ui/src/app/base/components/FormikField/FormikField.tsx

```diff
@@ -10,7 +10,7 @@ import type { TSFixMe } from "app/base/types";
 type Props = {
   Component?: JSX.Element;
   name: string;
-  value?: number | string;
+  value?: number | string | readonly string[];
```

Comment:
Could be worth a comment as to why this might be readonly.

Reply:
I've opted to use `HTMLProps<HTMLElement>["value"]` instead which I think is clearer what `value` is supposed to/can be. `FormikField` would probably be better off using a generic to determine what props are allowed but I think that might need a bit more thought.